### PR TITLE
Use tagName for bsy-switch component

### PR DESCRIPTION
**Pull request by JovicaSusa**
This should handle #96

### CODE REVIEW

File: assets/stylesheets/bookingsync/_ember-toggle.scss [hunk 1]
```diff
@@ -7,3 +7,7 @@
 .x-toggle:checked + label > .x-toggle-light.x-toggle-btn {
   background: $brand-primary;
 }
+
+.bsy-switch {
```
Comment (herzzanu) [inline assets/stylesheets/bookingsync/_ember-toggle.scss]: Why do we need this? :)

Comment (JovicaSusa) [reply assets/stylesheets/bookingsync/_ember-toggle.scss]: without that styling is broken, switch can't be inline with other elements.

**Comment by thilonel** [comment]
@JovicaSusa @herzzanu labels!

**Comment by thilonel** [comment]
Wow not even labels, this is a PR in waves 😱

**Comment by JovicaSusa** [comment]
@thilonel It looks I've added PR instead of issue by mistake, sry.

**Pull request merged**